HW05.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+title: "HW05"
+author: "Nabilah Sammudin"
+date: "8/4/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+### PART 1
+
+## 1. Tidying the dad_mom file
+
+In this repo, you'll find a dad_mom.txt file that isn't particularly tidy. Clean this up so there are only 4 columns 
+Tidy this data frame so that it adheres to the tidy data principles:
+
+    Each variable must have its own column.
+    Each observation must have its own row.
+    Each value must have its own cell.
+    
+```{r} 
+#load library
+library(knitr)
+library(tidyverse)
+library(magrittr)
+```
+First of all, read in the .txt file and print it out
+```{r}
+dad_mom <- read.table("dad_mom.txt", header = TRUE) #read in .txt file
+dad_mom #print data frame to see values
+```
+The table needs to be cleaned such that there are only 4 columns. Determine the 4 columns to be family, name, parent and income. 
+```{r}
+tidy_dad_mom <- dad_mom %>% #tidy data
+  unite (Dad, name_dad, income_dad) %>% #unite data for dads and moms
+  unite (Mom, name_mom, income_mom) %>%
+  gather (key = "Parent", value = "name_income", -fam_id) %>% #gather data
+  separate(name_income, into = c("Name", "Income")) %>% #separate column names
+  rename(Family = fam_id) #rename fam id column so it looks neater
+tidy_dad_mom #print data frame
+
+kable(tidy_dad_mom, format = "markdown", align=rep('c')) #to view/print table using kable
+```
+
+### PART 2
+Goal: To join the 5 .csv files in CTRP together 
+
+First of all, read in all files
+```{r}
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv") #read in all .csv files
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+cancer_cell_line_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+compound_info <- read.csv("CTRP_files/compound_info.csv")
+experiment_info <- read.csv("CTRP_files/experiment_info.csv")
+
+#glimpse each file in console to identify variables that can be combined
```

Comment:
I like how you added the comment here so we get a sense of all the steps that you went through while doing the assignment!  Also really nice job with the comments this time around, I know that was something you mentioned wanting to work on and I definitely see you doing that here!